tests/e2e/vscode-settings/specs/use-vscode-settings.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from "@wdio/globals";
+import { VSCodePO } from "../../page-objects";
+
+describe(".vscode/settings.json", () => {
+    it("should use config path from user settings", async () => {
+        const vscodePO = await VSCodePO.create();
+        const testingViewControl = vscodePO.getTestingViewControl();
+        await testingViewControl.open();
+
+        const sidebar = vscodePO.getTestingSideBar();
+        await sidebar.waitTestsRead();
+
+        const [firstSection] = await sidebar.getSections();
+        const [mainTreeItem] = await firstSection.getVisibleItems();
+
+        await mainTreeItem.expandAll();
+        const testsFullTitle = await mainTreeItem.getTestsFullTitle();
+
+        expect(testsFullTitle).toEqual(["tests test.testplane.ts success another-bro"]);
```

Comment:
check that used browser name from config that specified in `.vscode/settings.json`

Reply:
Weak test, imho
I would just break default testplane config, and here tested that testplane config is successfully read